lib/MusicBrainz/Server/Report/RecordingsWithTheSameNameByDifferentArtistsWithTheSameName

```diff
@@ -0,0 +1,37 @@
+package MusicBrainz::Server::Report::RecordingsWithTheSameNameByDifferentArtistsWithTheSameName
```

Comment:
This needs to be shorted (here and elsewhere) like how we do with the `ArtistsDisambiguationSameName` report, to something like `RecordingsSameNameDifferentArtistsSameName`, or shorter if @yvanzo can think of something.